src/Workspaces/Core/Portable/Utilities/NullableHelpers/TypeSymbolWithNullableAnnotation.cs

```diff
@@ -187,7 +187,9 @@ public ImmutableArray<SymbolDisplayPart> ToMinimalDisplayParts(SemanticModel sem
 
             public ImmutableArray<SymbolDisplayPart> ToMinimalDisplayParts(SemanticModel semanticModel, int position, SymbolDisplayFormat format = null)
             {
-                return WrappedSymbol.ToMinimalDisplayParts(semanticModel, position, format);
+                // Call the right API once https://github.com/dotnet/roslyn/pull/35698 is merged
```

Comment:
i trust you here :)

Reply:
I'm not sure I trust myself. This is my fault for not getting the compiler API in earlier, but at this point that's locked down for this release and for _that_ PR doesn't meet the bar. This is still worth fixing though.

Reply:
Why are we using flow state at all? Why can't this just take a NullableAnnotation?

Reply:
We forgot to add the overload, and now the compiler is locked and adding it (which I did in that other PR but then didn't merge) will take too much work. So I'll replace this with the real implementation in 16.4.